optaplanner-examples/src/main/java/org/optaplanner/examples/batchscheduling/domain/Batch.java

```diff
@@ -0,0 +1,49 @@
+package org.optaplanner.examples.batchscheduling.domain;
+
+import java.util.List;
+
+import org.optaplanner.examples.common.domain.AbstractPersistable;
+
+import com.thoughtworks.xstream.annotations.XStreamAlias;
+
+@XStreamAlias("PipeBatch")
+public class Batch extends AbstractPersistable {
+
+    private List<RoutePath> routePathList;
+    private String name;
+    private Double volume;
+    private Long delayRangeValue;
```

Comment:
Suggest to use primitive types, unless you need these nullable for some reason. (See similar comment below.)